src/utils/IConfig.ts

```diff
@@ -0,0 +1,32 @@
+import * as AWSXRay from "aws-xray-sdk";
+import {SecretsManager} from "aws-sdk";
+import {GetSecretValueRequest, GetSecretValueResponse} from "aws-sdk/clients/secretsmanager";
+import {safeLoad} from "js-yaml";
+
+export interface IConfig {
+  azure: {
+    tennant: string
+    appId: string
+    issuer: string
+    jwk_endpoint: string
+  };
+}
+
+export async function getConfig(): Promise<IConfig> {
```

Comment:
Probably doesn't belong in an interface class

Reply:
I've split them in d3aff9f